cloud/kubernetes/helm/expose-all.yaml

```diff
@@ -16,3 +16,8 @@ serviceEndpoints:
     app: "yb-tserver"
     ports:
       yedis-port: "6379"
+
+gflags:
+  tserver:
+    use_cassandra_authentication: True
```

Comment:
Lets keep the default here false. We can just add docs to try setting this to true.